includes/class-wc-stripe-webhook-handler.php

```diff
@@ -1505,7 +1505,7 @@ private function get_order_from_intent( $intent ) {
 				$order = isset( $data[0], $data[1] ) ? wc_get_order( absint( $data[0] ) ) : false;
 
 				if ( $order ) {
-					$intent_id = WC_Stripe_Helper::get_intent_id_from_order( $order );
+					$intent_id = WC_Stripe_Order_Helper::get_intent_id_from_order( $order );
 
 					// Return the order if the intent ID matches.
 					if ( $intent->id === $intent_id ) {
```

Comment:
There are a few order-related methods in `WC_Stripe_Helper` that have not been moved to the new class and are being used in this file.
- `get_order_by_charge_id`
- `get_order_by_source_id`
- `get_order_by_refund_id`
- `get_order_by_intent_id`

Are these going to be moved later? Or are they intentionally not moved because they are not retrieving/updating any property from/in the order?